spec/kitchen_spec/page_element_spec.rb

```diff
@@ -88,13 +86,18 @@
       let(:metadata) { '' }
 
       it 'finds the title' do
-        expect(page1.title.text).to eq page_title_text
+        expect(page1.title.text).to eq 'Page title'
       end
     end
 
     context 'with metadata' do
-      it 'finds the title' do
-        expect(page1.title.text).to eq page_title_text
+      it 'skips metadata to find title' do
+        expect(page1.title.text).to eq 'Page title'
+      end
```

Comment:
test updated to catch whether `PageElement#title` is finding metadata title or actual page title